src/SprykerEco/Zed/Payone/Business/Api/Response/Container/AbstractResponseContainer.php

```diff
@@ -217,7 +217,7 @@ protected function get($name)
      *
      * @return bool|null
      */
-    protected function set($name, $value)
+    protected function set(string $name, $value): ?bool
```

Comment:
Same here.

Reply:
It was made for fix code-sniffer validation issues.

Reply:
**I've taken into account not to make fixes based solely on silencing checks.**
Regarding to this place, I disagree with you. PhpStan says me: bad thing, this code can return `null`. I answer: Okey, I can help you, lets add right type hint **because this code really returns null and it mentions in phpdoc as well.** Why can't I add right type hint if it's not an api? If it's better to change code to such way when we have only bool - it's cool, okay, but I didn't have goal to do deep refactoring. I can agree if it's case, when it's better to use another approach (f.ex. use cast instead of *orFail()). But for this place I've done what I should have. Now, for review step, we can consider should it be as it is or we can refactored it, but initially I did the right adjustment, I think (added correct type hint according to the code inside).

Okey, sure, if it's not a major and it's better to exclude null, I will change it.

Reply:
phpstan is just a "dumb" tool, a static analyzer, all it does is tell you that something does not match, it does not have an understanding of *why* it doesn't match. So just doing the change that it demands is not enough.

> Why can't I add right type hint if it's not an api?

Because what is right needs to be determined by a human, not phpstan. :) `?bool` was not right here. phpstan is a great tool to find bugs, but the code must be still checked.